apps/devhub/templates/devhub/versions/list.html

```diff
@@ -310,7 +310,7 @@ <h3 data-tmpl="{{ _('Delete Version {version}') }}"></h3>
         {% endtrans %}
       </p>
       <p class="beta-warning"><br /><em>{{ _('Warning:') }}</em>
-        {% trans bugzilla_url='https://bugzilla.mozilla.org/enter_bug.cgi?product=addons.mozilla.org&component=Developer%20Pages' %}
+        {% trans url='https://github.com/mozilla/olympia/issues' %}
```

Comment:
This should stay as `bugzilla_url` or you should update `{{ bugzilla_url }}` to be `{{ url }}` below.